_meta/kibana/6/dashboard/apm-dashboards.json

```diff
@@ -592,7 +592,7 @@
               "type": "top_hits",
               "schema": "metric",
               "params": {
-                "field": "context.service.name",
+                "field": "service.name",
```

Comment:
how does this work with aliases?  Shouldn't block this pr but I think it needs a follow up to use either `service.name` or `context.service.name`, which is present.

Reply:
Good point, I created a follow up issue https://github.com/elastic/apm-server/issues/1786.